_specs/ecip-1064.md

```diff
@@ -0,0 +1,135 @@
+---
+lang: en
+ecip: 1064
+title: Communicating with 3rd parties
+status: Draft
+type: Informational
+author: Dean Pappas (deancpappas@gmail.com)
+created: 2019-08-06
+---
+
+## Abstract
+
+A process for communicating high priority updates and events to 3rd parties 
+
+## Motivation
+
+One of the biggest issues for coordinated decentralized network upgrades (ie hard fork) is disseminating this information to 3rd parties. As it stands, multiple entities can be contacting 3rd party organizations as “official” representatives, recommending different clients and actions, or even publishing conflicting information that can lead to centralization, network splitting, and other systemic issues caused by this lack of coordination. 
+
+There can also be new processes that come out as a result of this, ie a “safe list” of 3rd party organizations. It is not the purpose of this ECIP to outline these.
+
+
+## Specification
+
+This process is comprised of 3 parts
+
+1) Representative Selection
+2) Message Construction
+3) Channels
+
+### Representative Selection
+
+Each Client (geth/multigeth/parity/mantis) should have 1 representative selected to help in the outreach process.  In addition, there should be 2 representatives voted by the Client representatives that are unaffiliated with any of the Organizations.
+
+This selection happens at the “Final Call” of the event.
```

Comment:
Maybe specify that this only applies to "Final Call" of Hard Fork Meta ECIPs. ECIP has a status called "Final Call", and I don't think all of them will need to go through this process.

Reply:
Totally agree

Reply:
@bobthegrownup preferably before we merge it. Let me know if you need help with that.

Reply:
@sorpaas @YazzyYaz I added the Hard Fork detail for ECIPs and updated the proposal with the concept of "Black Swan" events. The focus is still with hard forks, but it now encompasses the emergency events.